sample-app/src/components/FilterSearch.tsx

```diff
@@ -0,0 +1,92 @@
+import { FilterSearchResponse } from "@yext/answers-core";
+import { SearchParameterField } from "@yext/answers-core";
+import { useRef, useState } from "react";
+import { useAnswersActions } from '@yext/answers-headless-react';
+import InputDropdown from "./InputDropdown";
+import renderWithHighlighting from "./utils/renderWithHighlighting";
+import { Option } from "./Dropdown";
+
+const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+
+export interface FilterSearchProps {
+  title: string,
+  sectioned: boolean,
+  searchFields: SearchParameterField[],
+  screenReaderInstructionsId: string
+}
+
+export default function FilterSearch (props: FilterSearchProps): JSX.Element {
+  const { title, sectioned, searchFields, screenReaderInstructionsId } = props;
+  const answersActions = useAnswersActions();
+  const [input, setInput] = useState('');
+  const [results, updateResults] = useState<FilterSearchResponse|undefined>();
+  const [, setMessage] = useState( '' );
+  const requestId = useRef(0);
+  const responseId = useRef(0);
+
+  async function executeFilterSearch (inputValue: string) {
```

Comment:
I think we can use the `useAutocomplete` hook but maybe call it `useSynchronizedSearch` (probably a better name) and takes in a function to execute either autocomplete or filtersearch.

Reply:
abstracting out this logic so that we don't duplicate it makes sense to me